tvos/plugin/src/android/index.js

```diff
@@ -54,7 +54,7 @@ export class AndroidPlayer extends React.Component<Props, State> {
     DeviceEventEmitter.addListener("emitterOnError", this.onError);
     DeviceEventEmitter.addListener("onTvKeyDown", this.onKeyDown);
     DeviceEventEmitter.addListener("onShowSettings", this.onShowSettings);
-    sendQuickBrickEvent("blockTVKeyEmit", { blockTVKeyEmit: false });
+    sendQuickBrickEvent("blockTVKeyEmit", { blockTVKeyEmit: true });
```

Comment:
Why do we need to change this to true?

Reply:
this i part of marge from 0.6.5 to the master, it wasn't marge